pkg/riff/commands/root.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/projectriff/riff/pkg/cli"
+	"github.com/spf13/cobra"
+)
+
+func NewRootCommand(c *cli.Config) *cobra.Command {
+	cmd := NewRiffCommand(c)
+
+	cmd.Use = c.Name
+	if c.GitDirty == "" {
```

Comment:
`GitDirty` can be a boolean then

Reply:
Can you set a bool with ldflags?

Reply:
Ah ah, good point

Reply:
On second thought, we can make this a bool, since it's derived from the ldflag set value and not the literal pointer.